src/gsy_e/models/strategy/area_agents/settlement_agent.py

```diff
@@ -1,3 +1,20 @@
+"""
+Copyright 2018 Grid Singularity
+This file is part of D3A.
```

Comment:
Reminder to update the newly added modules
```suggestion
This file is part of Grid Singularity Exchange.
```